ush/ufsda/disk_utils.py

```diff
@@ -16,6 +17,41 @@ def mkdir(dirpath):
         logging.info(f"{dirpath} could not be created")
 
 
+def removefile(file):
```

Comment:
Why are some of these shutil and then the first one just `os.remove`?

Reply:
Lines 49-50 should be identical to 24-25, no?

Reply:
good catch @CoryMartin-NOAA ... and `shutil.rm` doesn't exist ...

Reply:
@CoryMartin-NOAA , made the changes you suggested